### PR TITLE
Add and removing book list

### PR DESCRIPTION
**Pull request by HSMathebula**
### Awesome-books

in this project I:

- Add HTML file
- Add style.css
- Add book.js module
- Add declaration.js module
- Add bookData.js module
- Add luxon.js module
- Add index.js

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
+body {
+  text-align: center;
+}
+
```
Comment (ShahierNashaat) [inline style.css]: - [ ] Please change the file name from `style.css` to `index.css`.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @HSMathebula , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/bookData.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          <button class="deleteBtn" id="${StoredBooks[i].id}">
+          remove</button>
+          </div>
+        `;
+  }
+
+  document.querySelectorAll('.deleteBtn')
+    .forEach((btn) => {
+      btn.addEventListener('click', () => {
+        deleteData(btn.id); // eslint-disable-line
```
Comment (youmari) [inline modules/bookData.js:29]: - [ ] [OPTIONAL] it would be better to use object destructuring here

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @HSMathebula 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**